src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -141,7 +142,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String messageContent = request.getParameter("message");
 
     // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    // altered the whitelist so that text modifiers aren't removed
+    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.basic());
+    System.out.println(cleanedMessageContent);
```

Comment:
You probably don't need print statements in the final version

Reply:
yep definitely. These were a remnant from me trying to figure out the issue with Jsoup adding random newlines.